Google.PowerShell.IntegrationTests/SQL/SQL.GcSqlInstances.Tests.ps1

```diff
@@ -211,6 +211,29 @@ Describe "Import-GcSqlInstance" {
         { Import-GcSqlInstance "test-db2" "gs://gcsql-csharp-import-testing/testsql.gz" "newguestbook" } | Should Throw `
         "ERROR 1840 (HY000) at line 24: @@GLOBAL.GTID_PURGED can only be set when @@GLOBAL.GTID_EXECUTED is empty."
     }
+
+    It "should import a local file and delete the bucket for a local file upon completion" {
```

Comment:
"delete the bucket" makes no sense out of context. How about "import a local file by uploading it to GCS"?